stylus/utilities/text.styl

```diff
@@ -30,6 +30,20 @@ $ellipsis
     overflow hidden
     text-overflow ellipsis
 
+spacellipsis()
+    overflow hidden
+    text-overflow ellipsis
+    white-space break-spaces
+
+space-ellipsis = {
+    'spacellipsis': 'spacellipsis'
+}
+
+if cssmodules == true
+    cssModulesUtils(space-ellipsis, breakpoints)
+else
+    nativeUtils(space-ellipsis, breakpoints)
+
```

Comment:
Do we need this class to be available with media queries? Since the other text utilities aren't.

Reply:
Oui c'est typiquement le besoin ici, d'avoir le spacellipsis uniquement sur mobile